resources/views/posts/_form.blade.php

```diff
@@ -106,6 +106,8 @@
 @endpush
 
 @push('scripts')
+    <script src="{{ asset('theme/jquery.min.js') }}"></script>
+
     <script type="text/javascript" src="https://unpkg.com/trix@2.0.0/dist/trix.umd.min.js"></script>
```

Comment:
since a build is being used it would be better to have these install in the build rather then using a CDN